LRBv21.py

```diff
@@ -231,15 +231,17 @@ def iterate(cvar, Tu):
             cvar = cz0_guess
             
         elif control_variable == "density":
-            # Initialise control variable as the starting density
+            # Initial guess of nu0 assuming qpll0 everywhere and qpll=0 at target
             # Impurity fraction is set to constant as cz0
-            cvar = nu0
+            nu0_guess = np.sqrt((qpllu0**2 ) /(2*kappa0*cz0 *Tu**2 *integralinterp(Tu)))
+            cvar = nu0_guess
         
         elif control_variable == "power":
-            # If control variable = Power, keep nu0 and cz0 constant
-            # Initialise control variable 1/qradial. This is needed so that
+            # Initialise from Lengyel
             # too high a cvar results in a positive error!
-            cvar = 1/qradial #qpllu0
+            qpllu0_guess = np.sqrt(2*kappa0*cz0*nu0**2*Tu**2*integralinterp(Tu))
```

Comment:
I think this qpllu0_guess should just be qpllu0 from input

Reply:
Thanks, this makes sense. I've put in a commit and a comment to this effect.